server/db/models/wishlist.js

```diff
@@ -0,0 +1,15 @@
+//define Wishlist model here
+const Sequelize = require('sequelize');
+const { db } = require('./../database.js');
```

Comment:
the bracket notation is not needed here, I believe:

**const db  = require('./../database.js');**